### DISCUSSION TITLE
Change default storage location

### DISCUSSION THREAD

**Discussion by LoganJFisher** [mealie-recipes/mealie#2987] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

As Mealie can only save locally, this creates problems when local space is limited.

### Please provide a concise description of the feature that would resolve your issue.

I would like to be able to store my recipes in a different location than the default. In particular, I have a NAS, but have Mealie running on an RPi4. I'd like to relocate my recipes to the NAS, which Mealie then accesses over my network.

### Please consider and list out some caveats or tradeoffs made in your design decision

Nothing comes to mind.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by hay-kot** [comment]
If you're using Postgres you can mount the volume of the container to your shared NAS drive that is mounted to the host.

If you're using SQLite there isn't a way to do this, as SQLite does not support being stored of network drives as far as I'm aware.

**Reply by LoganJFisher** [reply]
I'm using the Home Assistant add-on.

**Reply by hay-kot** [reply]
We don't provide any official Home Assistant add-ons so you'll have to talk to the maintainer of that project.
